apps/web/vibes/soul/sections/products-list-section/index.tsx

```diff
@@ -108,7 +108,7 @@ export function ProductsListSection({
                   />
                 )}
               </Stream>
-              <aside className="block @3xl:hidden">
```

Comment:
This is intended, right?

Reply:
Yes, see first commit: 97631bc8a78aa0eef9e14bb2ca695a14f6761e47